test/test_ecr_utils.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from unittest import mock
+from dcicutils.ecr_utils import ECRUtils
+from dcicutils.docker_utils import DockerUtils
+
+
+REPO_URL = '123456789.dkr.ecr.us-east-2.amazonaws.com/cgap-mastertest'
```

Comment:
I assume this is a sample and not a real one? Maybe a comment saying not to expect it to work in integration tests.